src/commands/lm/info.js

```diff
@@ -0,0 +1,42 @@
+const { Command } = require('@oclif/command')
+const Listr = require('listr')
+const chalk = require('chalk')
+const {
+  GitValidators,
+  checkLFSFilters,
+  checkHelperVersion
+} = require('../../utils/lm/requirements')
+
+class LmInfoCommand extends Command {
+  async run() {
+    const steps = GitValidators
+    steps.push(
```

Comment:
This would modify `module.exports.GitValidators`, which means another module importing it would see this mutation.
Instead, using array spread should create a new array, so the original imported value is not modified.

```js
const steps = [
  ...GitValidators,
  {
     title: '...'
  // And so on
]  
```

Reply:
thanks, updated!